.gitignore

```diff
@@ -8,3 +8,5 @@
 /chorus-js.iml
 /chorus-js.ipr
 /chorus-js.iws
+/.idea/
+/yarn.lock
```

Comment:
If using yarn, the yarn lock file should be committed to the repository.
This is to ensure anybody running `yarn` to install the js dependency gets **exactly** the same set of dependencies.

However, it looks like all the commands here are assuming `npm` so if we're not using yarn, how did that end up here?

Reply:
Good spot I have removed it